docs/how-to-guides/feathr-mlops.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Using Feathr with MLOps v2
+parent: How-to Guides
+---
+
+# Using Feathr with MLOps v2
+
+MLOps v2 Solution Accelerator is a starting point for MLOps implementation in Azure. Prerequisites, documentation, and other important information can be found in the [Feathr branch of MLOps v2 Accelerator](https://github.com/Azure/mlops-v2/tree/feature/feathr). Feathr can be used in MLOps v2 accelerator, together with Terraform as infrastructure-as-code tool and Azure Machine Learning (AML) for training/deploying the endpoint.
```

Comment:
Should `https://github.com/Azure/mlops-v2/tree/feature/feathr` be updated to use main branch?

Reply:
It is currently not possible to merge branches in the MLOps v2 repo because the scripts in the Feathr branch are different and aren't really compatible with what they originally had. 

We talked about the possibility of making a whole separate folder for Feathr and merging branches, but Ritaja said their current system relies on that exact folder structure so they would have to negotiate reworking the whole structure. I will bring it up again but currently we can only work with the `feature/feathr` branch

Reply:
Thanks Rizo, I will merge your PR, thanks for contribution!